lib/feature_flagger/control.rb

```diff
@@ -25,18 +27,20 @@ def releases(resource_name, resource_id)
     end
 
     def release_to_all(feature_key)
+      @notifier.send(FeatureFlagger::Notifier::UNRELEASE_TO_ALL, feature_key)
```

Comment:
The constant seems to be wrong
```suggestion
      @notifier.send(FeatureFlagger::Notifier::RELEASE_TO_ALL, feature_key)
```